EpiResponsivePicture/TagBuilders/PictureTagBuilder.cs

```diff
@@ -87,11 +87,12 @@ public TagBuilder Build()
 
     private void SetPictureData()
     {
-        var imageFound = ServiceLocator.Current.GetInstance<IContentLoader>().TryGet<IContentData>(pictureContentReference,
-            new LoaderOptions { LanguageLoaderOption.FallbackWithMaster() }, out var content) && content is IImage or IResponsiveImage;
+        var imageFound = ServiceLocator.Current.GetInstance<IContentLoader>().TryGet<IContentData>(
+            pictureContentReference,
+            new LoaderOptions { LanguageLoaderOption.FallbackWithMaster() }, out var content);
 
         if(imageFound)
```

Comment:
how about:
```
if(imageFound && content is IImage image)
            SetImageAltText(image);
```

In this case changes you made in SetImageAltText method are not required as this method will not be called if image is not implementing IImage